internal/test/wmcb/runTest.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+cd wmcb
```

Comment:
You need to call out that  this cannot be executed locally and this script can only be run within the container.